### ISSUE TITLE
Some color contrast issues

### ISSUE THREAD

**Issue by vperezcadavid99** [phetsims/make-a-ten#334]
**Test device**
Choromebook
**Operating System**
ChromeOS
**Browser**
Chrome
**Problem description**
From https://github.com/phetsims/qa/issues/1419

Some color contrast issues:

- During the game screen, the blue levels fail the color contrast test against background:
Blue level icon (#59A2AB) Vs Background (#ECFEB9) -----> 2.7:1

<img width="260" height="216" alt="Image" src="https://github.com/user-attachments/assets/c04764bb-3ed0-4a24-a292-8ce850b1da02" />

- On the Adding and Explore screens, the number cards fail the color contrast test against the background. The gray border does not give enough contrast to clearly distinguish the cards from the background.

Number card (#F2F3EF) Vs Background (#ECFEB9) ----->1.03:1
<img width="260" height="216" alt="Image" src="https://github.com/user-attachments/assets/5ec19df2-d592-4da3-9a68-9b2442532da2" />

-  On the Explore screen, the light blue panel fails the color contrast test against the number cards. The gray border does not give enough contrast to clearly distinguish the panel from the cards.

Number card (#F2F3EF) Vs Background (#D4DEED) ----->1.21:1
<img width="350" height="216" alt="Image" src="https://github.com/user-attachments/assets/789f0b87-e134-4df3-a054-c868c4fab992" />

- On the Game screen, the "Back to Menu" button fails the color contrast test against the background. The button also fails when the border is included in the contrast test for all items:

back to menu button (#F4E14F) Vs Background (#DE5380) -----> 2.78:1
<img width="340" height="190" alt="Image" src="https://github.com/user-attachments/assets/f00dd47b-bf57-423d-9c93-86fdf03ce5d8" />

back to menu button (#F4E14F) Vs Background (#5AA3AB) -----> 2.16:1
<img width="340" height="190" alt="Image" src="https://github.com/user-attachments/assets/c678a346-42e1-4c97-866b-c770f54b508a" />

back to menu button (#F4E14F) Vs Background (#927AC6) -----> 2.7:1
<img width="340" height="190" alt="Image" src="https://github.com/user-attachments/assets/1c76d4bb-e81a-4b0c-8f4b-eb999872aa3e" />

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Make a Ten‬
URL: https://phet-dev.colorado.edu/html/make-a-ten/1.1.0-rc.1/phet/make-a-ten_all_phet.html
Version: 1.1.0-rc.1 2026-07-20 18:46:27 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/150.0.0.0 Safari/537.36
Language: en-US
Window: 1600x812
Pixel Ratio: 1.7999999523162842/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}
<details/>

**Comment by catherinecarter** [comment]
Thanks, @vperezcadavid99. 

I also noticed these issues when checking color contrast ratios.

For the game: 
* The game levels have a (albeit thin) black border around the tiles, which does pass color contrast
* The 'back' button is black, so while the button itself may disappear, the black arrow is sufficient (verified on Slack: @arouinfar said: "For buttons, the relevant contrast is the icon against the button background. The button background to sim background doesn't matter."

For the paper ones:
* Similar to the back button, the pedagogical importance are the numbers themselves, which are black
* While the paper does disappear, the number and the handle are visible in both instances, so this is ok as three cues are given (the black color of the number, the number itself, and the handle). 
<img width="541" height="310" alt="Image" src="https://github.com/user-attachments/assets/c801d835-3a01-49f0-9c42-6ad733fae70f" />
* It's a little awkward to see a floating handle, and would be nice to see the card, when I played around with increasing the contrast of the background, it looked pretty distracting and made the digits difficult to see (the most important part of the sim), especially for numbers bigger than 10 (but not a multiple of 10)
<img width="267" height="257" alt="Image" src="https://github.com/user-attachments/assets/f68b921a-8d8f-44dc-afac-8c96cf7554b6" />


So I think all of these color contrast failures are ok. Thank you for double checking, it is much appreciated.

Closing.
